pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -313,6 +313,80 @@ self: super: {
     '';
   });
 
+  coq_nvim = super.coq_nvim.overrideAttrs (old: {
+    passthru.python3Dependencies = let
+
+      # Extract the version of the dependencies from the requirements.txt file
+      requirements = builtins.readFile "${old.src}/requirements.txt";
+      requirementsLines = lib.splitString "\n" requirements;
+
+      extractLine = pname: lib.findSingle
+        (lib.strings.hasPrefix pname)
+        "none"
+        "multiple"
+        requirementsLines;
+
+      extractDepVersion = pname:
+        let
+          line = extractLine pname;
+          hashPart = lib.lists.last (lib.splitString "/" line);
+          hash = lib.strings.removeSuffix ".tar.gz" hashPart;
+        in hash;
+    in ps: with ps; [
+      pynvim
+      pyyaml
+      (let
+        version = extractDepVersion "pynvim_pp";
+
+        # This maps the current version of pynvim_pp to the sha256 of its source.
+        # Hence, if the dependency version changes in the requirements.txt, the build will fail.
+        version2Hash."456765f6dc8cef6df39ae3e61c79585258d38517" = "sha256-edDDzxR60ILFbv1CnYGxW/sJDMeRIFPjmTeiMRdvA3k=";
```

Comment:
Basically this.

But you can pass directly to the hash parameter like

```
hash = { *commit* = "*hash*";}."${parsedCommit}"
```

Reply:
Done !